test/mock-dom.js

```diff
@@ -0,0 +1,8 @@
+global.window = {};
+global.document = {
+	createElement: function() {
+		return {
+			tagName: 'iframe'
+		};
+	}
+};
```

Comment:
The window and document objects need to be defined before iframe-resizer was imported, and this was the only way I could figure out how to do it..